packages/mui-lab/src/SnackbarsProvider/SnackbarsProvider.tsx

```diff
@@ -0,0 +1,280 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Snackbar, { SnackbarProps } from '@mui/material/Snackbar';
+import Grow from '@mui/material/Grow';
+import SnackbarContent from '@mui/material/SnackbarContent';
+import { styled, useTheme } from '@mui/material/styles';
+import SnackbarsContext, { ShowSnackbarProps, SnackbarContentType } from './SnackbarsContext';
+import SnackbarsContainer from './SnackbarsContainer';
+
+const randomId = () => `mui-${Math.round(Math.random() * 1e5)}`;
```

Comment:
Do we have a id generation `util` method that I can reuse here? I tried `useId`, it does not work here.